### PR TITLE
feat(docs-infra): simplify image styles and remove figure references

### PR DESCRIPTION
**Pull request by sjtrimble**
Reference #33259
Removes figures elements as AIO is not typically using captions or image groups where figures would be necessary or appropriate

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
Removes `figure` elements not necessary

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by gkalpak — APPROVED** [review]
❤️

File: aio/content/guide/docs-style-guide.md [hunk 1]
```diff
@@ -1267,27 +1267,27 @@ The doc generator reads the image dimensions from the file and adds width and he
...
+<div class="lightbox">
  <img src="generated/images/guide/docs-style-guide/flying-hero.png" alt="flying Angular hero" width="200">
-</figure>
+</div>
 
 ```html
 
-<figure>
+<div class="lightbox">
  <img src="generated/images/guide/docs-style-guide/flying-hero.png"
```
Comment (gkalpak) [inline aio/content/guide/docs-style-guide.md]:
```suggestion
  <img src="generated/images/guide/docs-style-guide/flying-hero.png"
```

File: aio/content/guide/docs-style-guide.md [hunk 2]
```diff
@@ -1267,27 +1267,27 @@ The doc generator reads the image dimensions from the file and adds width and he
...
    alt="flying Angular hero"
```
Comment (gkalpak) [inline aio/content/guide/docs-style-guide.md]:
```suggestion
     alt="flying Angular hero"
```

File: aio/content/guide/docs-style-guide.md [hunk 3]
```diff
@@ -1267,27 +1267,27 @@ The doc generator reads the image dimensions from the file and adds width and he
...
    width="200">
```
Comment (gkalpak) [inline aio/content/guide/docs-style-guide.md]:
```suggestion
     width="200">
```

**Comment by gkalpak** [comment]
@googlebot I consent.

**Comment by gkalpak** [comment]
**merge-assistance:** Global approval for this docs-only change.

**Comment by kara** [comment]
@gkalpak Are you giving this global approval for docs changes? I didn't see a comment

**Pull request closed without merging**